download/download.go

```diff
@@ -238,12 +244,14 @@ func formatItem(info map[string]string, repo, name string) ListItem {
 		Prefsize:     info["prefsize"],
 		Architecture: strings.ToUpper(info["arch"]),
 		Description:  info["Description"],
+		Timestamp:    timestamp,
 	}
 	item.Size, _ = strconv.Atoi(info["size"])
 
 	if repo == "apt" {
 		item.Version = info["Version"]
 		item.Architecture = info["Architecture"]
+		item.Size, _ = strconv.Atoi(info["Size"])
```

Comment:
do we have size only for deb packages? why not add this field to all artifacts?

Reply:
We have size field for template,raw files. We found bug only for apt

Reply:
ok